psi4/run_psi4.py

```diff
@@ -219,7 +219,7 @@
     tb_str += '\n'
     tb_str += type(exception).__name__
     tb_str += ': '
-    tb_str += exception.message
+    tb_str += str(exception)  # was exception.message so perhaps hybrid needed
```

Comment:
Whats wrong with message? We dont actually want the full string representation here as its overloaded.